# Autol LiDAR Frame.
std_msgs/Header          header
uint8 		lidar_index
AutolG32V2Packet[]   packets	    # array of packets
